apps/sms/views/shared/js/startup.js

```diff
@@ -126,7 +127,7 @@ var Startup = exports.Startup = {
 
       MessageManager.init();
       InboxView.init();
-      Navigation.init();
+      StartupManager.init(Navigation.init());
```

Comment:
maybe the issue comes from here: this should be `() => Navigation.init()` as argument